lib/phobos/listener.rb

```diff
@@ -87,13 +87,17 @@ def send_heartbeat_if_necessary
     private
 
     def listener_metadata
-      { listener_id: id, group_id: group_id, topic: topic, handler: handler_class.to_s }
+      { listener_id: id, group_id: group_id, topic: topics, handler: handler_class.to_s }
     end
 
     def start_listener
       instrument('listener.start', listener_metadata) do
         @consumer = create_kafka_consumer
-        @consumer.subscribe(topic, **@subscribe_opts)
+
+        # ruby-kafka does not provide an api to subscribe a group of particular topics
```

Comment:
There technically is one, although it would mean turning `topic` into a regex that includes all of the topics, e.g.
```ruby
topic_regex = Regexp.new("^#{topics.join('|')}$")
@consumer.subscribe(topic_regex, **@subscribe_opts)
```

...but I don't think that's any cleaner at all. :p Maybe just take the comment out.